CHANGELOG.md

```diff
@@ -105,6 +105,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Prime
 
+- Fixed: Crashes in several rooms when skipping cutscenes exactly 1 second from the end
+- Fixed: Incorrect shield texture for vertical **PowerBeamOnly** doors
+- Fixed: Elite Research - Stutter when loading room
+- Fixed: Ruined Shrine Access - Still playing Beetle fight music after leaving
+- Fixed: Research Entrance - Soflock if player kills pirates before touching the cutscene trigger (reverse labs)
+- Fixed: Mine Security Station - Wave Pirates not dropping down if the player didn't skip the cutscene immediately
+- Fixed: Uncrashed Frigate - Escape music sometimes playing at inappropriate times
+- Fixed: Furnace - Ghost elements re-appearing when re-entering from East Furnace Access
+- Fixed: Ruined Courtyard - Water platforms breaking when skipping cutscene on the last possible frame
+- Changed: Updated tournament winners scan
+- Changed: Ventilation Shaft: Cutscene skip no longer waits for load
+- Changed: Mine Security Station: Timing of activation of Wave Pirates is now the same regardless of what death animation the Shadow Pirates Play
+- Changed: Mine Security Station: Adjusted cutscene trigger so it can't be accidentally skipped
+- Changed: Ruined Shrine: Adjusted position of the cutscene skip lock-on point
+- Changed: Minor item acquisition jingle for health/pb/missile refill instead of major
+- Changed: Sun Tower Elevator: Rotated Samus Actor 90 degrees
```

Comment:
this reads confusing, the actor is only rotated during a cutscene i thought?

Reply:
Technically all of the cutscene elements were rotated relative to the Samus actor, but just mentioning the actor is simpler.